libkbfs/tlf_handle.go

```diff
@@ -522,15 +522,13 @@ func (h *TlfHandle) GetCanonicalName() CanonicalTlfName {
 	return h.name
 }
 
-func buildCanonicalPath(public bool, canonicalName CanonicalTlfName) string {
-	var folderType string
+// buildCanonicalPath returns a canonical path for a tlf.
+func buildCanonicalPath(public bool, tlfName CanonicalTlfName) string {
```

Comment:
It's still useful in a couple spots but is out of place. Renamed to `buildCanonicalPathForTlfName` and moved to path.go.